Libraries/Types/CoreEventTypes.d.ts

```diff
@@ -261,3 +261,23 @@ export interface PointerEvents {
   onPointerUp?: ((event: PointerEvent) => void) | undefined;
   onPointerUpCapture?: ((event: PointerEvent) => void) | undefined;
 }
+
+export interface NativeKeyEvent {
```

Comment:
this ain't macOS only to require // [macOS] ?

Reply:
It is, and also needs the diff tag.